app/views/layouts/map.html.erb

```diff
@@ -46,12 +46,17 @@
 
     <% unless @user %>
       <div class="welcome">
-        <h2><a><span class="icon close"></span></a><%= t 'layouts.intro_header' %></h2>
+        <h2><%= t 'layouts.intro_header' %></h2>
+        <div class="close-wrap"><span class="icon close"></span></div>
```

Comment:
That's cool, so long as it doesn't actually break any of those other close icons ;-)